frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

```diff
@@ -46,7 +46,7 @@ export default ({
     toggleWorksheetSize,
     showBundleContent,
 }) => (
-    <Sticky top={6} innerZ={1059}>
+    <Sticky top='#codalab-app-bar' innerZ={1059}>
```

Comment:
Can we actually factor out "codalab-app-bar" into a constant in constants.js? so that we don't actually refactor the header component and remove the id and make this break again

Reply:
Let's do this later